client/gutenberg/extensions/mailchimp/edit.jsx

```diff
@@ -182,36 +184,32 @@ class MailchimpSubscribeEdit extends Component {
 				</PanelBody>
 			</InspectorControls>
 		);
+		const blockClasses = classnames( className, {
+			[ `${ classPrefix }notication-audition` ]: audition,
```

Comment:
@jeffersonrabb FYI this was previously without `classPrefix` but I added it here for consistency since every other class has it via the variable. :-)